docker/bigdl-k8s/Dockerfile

```diff
@@ -98,6 +98,7 @@ ENV RUNTIME_DRIVER_MEMORY               10g
 ENV RUNTIME_PERSISTENT_VOLUME_CLAIM     myvolumeclaim
 ENV SPARK_HOME                          /opt/spark
 ENV JAVA_HOME                           /opt/jdk
+ENV BIGDL_CLASSPATH                     ${BIGDL_HOME}/jars/*
```

Comment:
so can we grep the jars here and set the BIGDL_CLASSPATH with all jars instead of using *? then the code in init k8s can be unchanged.

Reply:
assembly jars/ folder is a lib folder includes all the bigdl artifatcts without dependencies and all the common dependencies. If we do not using *, it requires adding all jars (>100 jars) as a long path.

Reply:
can we filter the jars by name start with bigdl?

Reply:
all the jars are needed, not only the bigdl jars.

Reply:
all these jars can be needed in the docker, but why do we need to set all of them to BIGDL_CLASSPATH?

Reply:
K8s image uses non-pip. Set all jars to BIGDL_CLASSPATH can make init_spark_on_k8s get it.

Reply:
but the other non-bigdl jars should be input into jars when spark-submit right? it doesn't make sense to put them in BIGDL_CLASSPATH?

Reply:
Actually, ${BIGDL_HOME}/jars/* is equal to `*-jar-with-dependencies.jar`. need to rename BIGDL_CLASSPATH?